### ISSUE TITLE
No more data since mid december 2022

### ISSUE THREAD

**Issue by maxdegraaf** [biemond/solaredge.modbus#24]
The SolarEdge App in combination with Homey worked just fine for a very long time. Not sure if it got broken because of an update on Homey or the App. I double checked all settings and nothing was change for sure. I can see in the App that the last data was received 22 days ago (mid december).
0efb9072-e2df-442c-8699-08e749b2a73e

**Comment by biemond** [comment]
yes I saw it .  connect and timeout issue.  Did you get some inverter update? 
I guess maybe ip has changed else remove the device and add it again. 

thanks

**Comment by maxdegraaf** [comment]
Found it. The DHCP server lost the reservation. Problem solved. Thanks!!!

**Comment by biemond** [comment]
cool
